### PR TITLE
Upgrade to RN 0.59.9 to support Xcode 11

### PR DESCRIPTION
**Pull request by LeoNatan**
.

### CODE REVIEW

**Comment by LeoNatan** [comment]
Test failure seems unrelated to RN upgrade in test suite.

File: detox/test/index.ios.js [hunk 1]
```diff
@@ -22,4 +22,5 @@ class exampleIos extends example {
   }
 }
 
+console.disableYellowBox = true;
```
Comment (d4vidi) [inline detox/test/index.ios.js]: @LeoNatan why is this needed? why do we all of a sudden have yellow boxes?

**Comment by d4vidi** [comment]
@LeoNatan did you also update the builds on jenkins (master and pr)?

File: detox/test/index.ios.js (hunk 1 above)
Comment (d4vidi) [reply detox/test/index.ios.js]: @LeoNatan I strongly advise against this

Comment (LeoNatan) [reply detox/test/index.ios.js]: This is only on iOS on debug. I don't need them.

Comment (LeoNatan) [reply detox/test/index.ios.js]: RN by default has yellowboxes on iOS 12 and 13.

**Comment by LeoNatan** [comment]
No, it's just in the project to have support for Xcode 11. CI is unchanged.
